CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- Add a customizable multi-window native display using egui ([#48])
```

Comment:
Sorry, you also need to define the link in line 16 :see_no_evil:

Reply:
:man_facepalming: uff